spec/lib/fix_maintainer_email_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe FixMaintainerEmail do
+  context 'maintainer' do
+    it 'should fix " at " to "@"' do
+      expect(FixMaintainerEmail.new('icesik at altlinux.org').execute)
+        .to eq('icesik@altlinux.org')
+    end
+
+    it 'should fix " dot " to "."' do
+      expect(FixMaintainerEmail.new('icesik@altlinux dot org').execute)
+        .to eq('icesik@altlinux.org')
+    end
+
+    it 'should fix "altlinux.ru" to "altlinux.org"' do
+      expect(FixMaintainerEmail.new('icesik@altlinux.ru').execute)
+          .to eq('icesik@altlinux.org')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.